user-guide/Getting_started/Installing_a_DMA/Using_a_pre_installed_DataMiner_VHDX.md

```diff
@@ -0,0 +1,155 @@
+   - *System URL*: This URL will grant you remote access to your DataMiner System web applications. You can choose to either [disable or enable this remote access feature](xref:Controlling_remote_access) at any time.
+   - *Admin Email*: This email is associated with the dataminer.services account that has the administrator role for the organization.
+   - *STaaS Region*: If you have selected to use [STaaS](xref:STaaS) for data storage, select the region where your data should be hosted.
+
+1. Verify the selected configuration and click *Configure*
+
+1. When the configuration is complete, click *Finish*
+
+   DataMiner will automatically start up, get licensed, and connect to dataminer.services. DataMiner Cube will also be installed, so you can connect to DataMiner locally.
+
+1. [Log in to DataMiner Cube](xref:Logging_on_to_DataMiner_Cube) using the previously configured Administrator account.
+
+   Logging in automatically with the built-in Administrator account is not possible, so you will need to fill in the username and password.
+
+> [!IMPORTANT]
+> For security reasons, we strongly recommend creating a second user and disabling the built-in administrator account once the setup is complete. See [Managing users](xref:Managing_users).
+
+## Switching from subscription mode to perpetual license
+
+When you deploy a DataMiner Agent using the pre-installed DataMiner Virtual Hard Disk, your system will run in subscription mode and get licensed automatically. Part of this process involves getting a DataMiner ID, which uniquely identifies your DataMiner Agent.
+
+If you have purchased a [permanent license](xref:Pricing_Perpetual_Use_Licensing), follow the steps below to convert your subscription installation to a perpetual-license one:
+
+1. [Stop the DMA using the DataMiner Taskbar Utility](xref:Starting_or_stopping_a_DMA_using_DataMiner_Taskbar_Utility).
+
+1. Open the *C:\Skyline DataMiner\\* folder.
+
+1. Remove all *\*.lic* files, if any.
+
+1. Open the *DataMiner.xml* file.
+
+1. Find the *&lt;DataMiner&gt;* tag and locate the *id* attribute.
+
+1. Note down the value in the *id* attribute.
+
+1. [Start the DMA using the DataMiner Taskbar Utility](xref:Starting_or_stopping_a_DMA_using_DataMiner_Taskbar_Utility).
+
+1. After a short while, a *Request.lic* file should appear in the `C:\Skyline DataMiner\` folder.
+
+1. Contact [dataminer.licensing@skyline.be](mailto:dataminer.licensing@skyline.be) and provide them with the ID and the *Request.lic* file.
+
+   In your email, mention that it concerns a conversion from a subscription to a perpetual license.
+
+1. Wait until you receive a *dataminer.lic* file from Skyline.
+
+1. When you have the *dataminer.lic* file, copy it to the `C:\Skyline DataMiner\` folder.
+
+1. [Start the DMA using the DataMiner Taskbar Utility](xref:Starting_or_stopping_a_DMA_using_DataMiner_Taskbar_Utility).
```

Comment:
This step occurs two times in this procedure, but DataMiner doesn't seem to be stopped in between. Is this correct?

Reply:
I believe it would be more correct to say to **re**start the DMA then.